test/api-test.js

```diff
@@ -137,4 +139,18 @@ describe('RippleAPI', function() {
       _.partial(checkResult, getOrdersResponse, done));
   });
 
+  it('getOrderBook', function(done) {
+    const orderbook = {
+      base: {
```

Comment:
Yeah:
```json
{
  "error": "remoteError",
  "error_message": "Remote reported an error.",
  "remote": {
    "error": "invalidParams",
    "error_code": 29,
    "error_message": "Invalid field 'taker_pays.issuer', not string.",
    "id": 1,
    "request": {
      "command": "book_offers",
      "id": 1,
      "ledger_index": "validated",
      "taker": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B",
      "taker_gets": {
        "currency": "USD",
        "issuer": null
      },
      "taker_pays": {
        "currency": "BTC",
        "issuer": null
      }
    },
    "status": "error",
    "type": "response"
  },
  "result": "remoteError",
  "engine_result": "remoteError",
  "result_message": "Remote reported an error.",
  "engine_result_message": "Remote reported an error.",
  "message": "Remote reported an error."
}
```

The `null` is there because of the json_rewrite on the issuer of undefined, which returns `NaN`, which serialises to null by JSON.stringify.

```javascript
>>> console.log(JSON.stringify({issuer: NaN}))
{"issuer":null}
```